doc/Datasources.rst

```diff
@@ -1,7 +1,7 @@
-Data Sources
+Data Source
```

Comment:
@nelsonam I think it will still build the docs but the asterisk under the word should be the same length as the word.